### ISSUE TITLE
Update user profile

### ISSUE THREAD

**Issue by tanerochris** [BuildForSDG/voxnostra#8]
Add functionality for a user to change the password and update personal information

**Comment by tanerochris** [comment]
Updates?

**Comment by mernxl** [comment]
Given #16 has functionalities to update user password, i guess when its merged we can add this feature.

Say on POST `user/{id}` given current user or administrator

If thats ok, Will provide as listed above.

**Comment by tanerochris** [comment]
yes, also add update user profile information, checkout the schema in #19
